### DISCUSSION TITLE
Health Check

### DISCUSSION THREAD

**Discussion by ad-momo** [adonisjs/core#4602] [Help]
Hello,

In [Adonis v6 migration doc](https://adonisjs.com/blog/future-plans-for-adonisjs-6#migrating-the-last-few-packages-to-adonisjs-v6)  there is the next sentence : "**Health checks - Removed from core to be its own package**." but I can't find it. 
Where is the documentation to implement HealthCheck ?

Thanks 😃 
Adrien

**Comment by MaximeMRF** [comment]
Hello @ad-momo,

It will be soon there, the v1 of the [new health check package](https://github.com/adonisjs/health) was release 15 minutes ago :)

**Reply by ad-momo** [reply]
Wow very nice ! 👍 

Thank you for reply 😃

**Comment by thetutlage** [comment]
Yeah, took a little longer to get all the APIs right. I will be announcing it on Saturday
